packages/clay-css/src/scss/variables/_date-picker.scss

```diff
@@ -51,7 +51,6 @@ $date-picker-nav-btn-monospaced: map-deep-merge(
 );
 
 $date-picker-nav-select: () !default;
-$date-picker-nav-select: map-deep-merge((), $date-picker-nav-select);
```

Comment:
Just a question about the unrelated this change:

- Double declaration in sequence: the last declaration rewrite the last declaration, right? why double declaration?
- Once you removed the last declaration, what's the expected side effect? if it exist, of course.

Reply:
@matuzalemsteles Not a problem, `map-merge` takes whatever was declared in the first map and combines the two. If there are two keys with the same name in both maps, whatever is declared in the second map wins between the two. This is a way (hack?) to provide default values with Sass maps.

An example that may help explain better:
```
// This one replaces `$map: () !default;` below it
$map: (
  background-color: #eee,
  color: red,
) !default;

$map: () !default;
$map: map-merge((
  color: black,
  font-size: 16px,
), $map);

// $map will be:
$map: (
  background-color: #eee,
  color: red,
  font-size: 16px,
);
```

If we don't use `map-merge` while providing default values:

```
$map: (
  background-color: #eee,
  color: red,
) !default;

$map: (
  color: black,
  font-size: 16px,
) !default;

// $map will be:
$map: (
  background-color: #eee,
  color: red,
);
```

The reason we don't need it here is because we aren't providing any default values for `$date-picker-nav-select`.

Reply:
Hmm makes sense, thanks for the clarification.